.github/workflows/ci.yml

```diff
@@ -69,7 +75,12 @@ jobs:
       - run: |
           python scripts/doc_checker.py asv_bench/benchmarks/utils.py \
             asv_bench/benchmarks/__init__.py asv_bench/benchmarks/io/__init__.py \
-            asv_bench/benchmarks/scalability/__init__.py
+            asv_bench/benchmarks/scalability/__init__.py \
+            modin/engines/base/io/column_stores \
```

Comment:
We can pass a whole directory to the script. If there is something to change here, fix it, please.

Reply:
Directories and files from `modin/engines/base/io` are passed into the script one by one because of `modin/engines/base/io/io.py` - this file need to be checked with additional arguments `--add-ignore=D102 --disable-numpydoc`.

Reply:
Okay, I see.